schema/attribute/birthdate.json

```diff
@@ -1,8 +1,10 @@
 {
-  "id": "http://platform.selfkey.org/schema/attribute/birthday.json",
-  "$schema": "http://platform.selfkey.org/schema/identity-attribute.json",
-  "title": "Birthday",
-  "description": "An individual's date of birth.",
-  "type": "string",
-  "format": "date"
+	"$id": "http://platform.selfkey.org/schema/attribute/birthday.json",
+	"$schema": "http://platform.selfkey.org/schema/identity-attribute.json",
+	"idAttribute": true,
+	"idAttributeRepository": "http://platform.selfkey.org/repository.json",
+	"title": "Birthday",
```

Comment:
Let's make this "Date of Birth" - we should stick to the formal name.

Let's also rename the file to `date-of-birth.json` for consistency.

Reply:
good idea

Reply:
fixed